graylog2-inputs/pom.xml

```diff
@@ -61,7 +61,7 @@
 
         <dependency>
             <groupId>org.apache.kafka</groupId>
-            <artifactId>kafka_2.9.2</artifactId>
+            <artifactId>kafka_2.11</artifactId>
```

Comment:
Wasn't a problem with the previous Kafka version obviously. Just want to make sure...